index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./index.css">
+    <title>Signup Page</title>
+</head>
+<body>
+    <nav>
+        <div>
+            <div>
+                Already have an Intuit account?<br/>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <button type="button">Sign In</button>
+        </div>  
+    </nav>
+    
+    <main>
+        <button type="button">
+            Privacy Settings
+        </button>
+        <div>
+            <img alt="intuit icon" src="./img/intuit-logo.jpeg">
+            <form>
+                <h3>Create an Intuit account</h3>
+                <p>One account for everything Intuit, including Mint.</p>
+                <p><a href="#">Learn more</a></p>
+                <div>
+                    <label for="email">
+                        <span>Email address</span>
+                        <input id="email" type="email">
+                    </label>
```

Comment:
you are using the wrong structure of a label input format, please change it into:
![image](https://user-images.githubusercontent.com/34653764/83829656-455d1c00-a6db-11ea-8f2c-c7320838943c.png)
for all the input labels.